pymc_experimental/tests/test_linearmodel.py

```diff
@@ -21,6 +21,7 @@
 import pandas as pd
 import pytest
 import xarray as xr
+from pytest import approx
```

Comment:
We usually use `np.testing.assert_allclose`, so it may be good to use it here also for consistency

Reply:
OK, good to know. I'll make that change.